transform/__init__.py

```diff
@@ -2,6 +2,8 @@
 
 app = Flask(__name__)
 
+from .views import test_views  # noqa
```

Comment:
Won't this import test_views whenever an app object is created? Might not be the best way to do this for production.